ui/imports/shared/stores/BIP39_en.qml

```diff
@@ -12,8 +12,10 @@ ListModel {
             if (xhr.readyState === XMLHttpRequest.DONE) {
                 root.words = xhr.responseText.split('\n');
```

Comment:
You could split directly with:
```js
```suggestion
                root.words = xhr.responseText.split(/\r?\n/);
```
without the extra check at the end